lib/dockerregistry/inventory_test.go

```diff
@@ -1993,23 +1995,18 @@ func TestCheckOverlappingEdges(t *testing.T) {
 
 type FakeCheckAlwaysSucceed struct{}
 
-func (c *FakeCheckAlwaysSucceed) Run(
```

Comment:
This stuff around edges should also be in a separate commit (I didn't check, can you verify?). I wonder if you rebased too far back into your history?

Reply:
Ping? This change is also in your other PR for image removal. You should only have it in 1 of them.